app/trip/components/trip-form.tsx

```diff
@@ -0,0 +1,67 @@
+import { useForm, FormProvider } from 'react-hook-form'
+import {
+  FormControl,
+  FormLabel,
+  // FormErrorMessage,
+  Image,
+  VStack,
+  HStack,
+  useColorModeValue
+} from '@chakra-ui/react'
+import { PrimaryButton } from '@/components/button'
+import { CustomDateTimePicker } from '@/components/customDateTimePicker'
+import { InputForm } from '@/components/input'
+import { TripFormTag } from '../components'
+
+export const TripForm = () => {
+  const methods = useForm()
+
+  const imageSrc = useColorModeValue(
+    '/images/no_image_light.jpg',
+    '/images/no_image_dark.jpg'
+  )
+
+  return (
+    <FormProvider {...methods}>
+      <VStack as="form" gap={{ base: '30px', md: '40px' }}>
+        <FormControl>
+          <FormLabel>Title</FormLabel>
+          <InputForm type="text" placeholder="Asakusa Temple" />
+        </FormControl>
+
+        <FormControl>
+          <FormLabel>Date From</FormLabel>
+          <CustomDateTimePicker
+            onChange={() => console.log('onChange')}
+            value={null}
+          />
+        </FormControl>
+
+        <FormControl>
+          <FormLabel>Date To</FormLabel>
+          <CustomDateTimePicker
+            onChange={() => console.log('onChange')}
```

Comment:
We might be able to clarify where `onChange` log comes from with stack trace on a browser, but I'm just wondering if it's worth logging out if you really need this log.

Reply:
This is just temporal... I will change it for the next logic PR